src/collection-view.js

```diff
@@ -634,22 +640,39 @@ const CollectionView = Backbone.View.extend({
   },
 
   // Render the child's view and add it to the HTML for the collection view at a given index, based on the current sort
-  addChildView(view, index) {
+  addChildView(view, index, options = {}) {
     if (!view || view._isDestroyed) {
       return view;
     }
 
+    if (_.isObject(index)) {
+      options = index;
+    }
+
+    // If options has defined index we should use it
+    if (options.index != null) {
+      index = options.index;
+    }
+
     if (!this._isRendered) {
       this.render();
     }
 
+    this._addChild(view, index);
+
+    // `preventRender` passed, so we are waiting another `addChildView`
```

Comment:
I don't think we should add opinion here in this comment.  There may be reasonable use cases we're not seeing apart from this. It simply means we're preventing `_renderChildren` as it does with `setComparator` and `setFilter`

Reply:
so, removing comment?
setComparator and setFilter has no comments

Reply:
either way for my tastes.  The var used is descriptive.

Reply:
no problem :)